### PR TITLE
feature(nosqlbench): Adding NoSQLBench 5.x to test Java v4 Driver

### PR DESCRIPTION
**Pull request by CodeLieutenant**
Adding docker image for `nosqlbench` to help java driver team to test 4.x versions
until `cassandra-stress` support for 4.x is ready to replace it.  

### Testing
- 🟢 [https://jenkins.scylladb.com/job/scylla-staging/job/dusan/job/longevity-10gb-3h-nosqlbench-test/21/](https://jenkins.scylladb.com/job/scylla-staging/job/dusan/job/longevity-10gb-3h-nosqlbench-test/21/)

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

**Review by fruch — DISMISSED** [review]
LGTM

**Review by vponomaryov — COMMENTED** [review]
The `style(nosqlbench): fixing precommit lints` commit must have been part of the first comimt -> `feature(nosqlbench): nosqlbench v5.x to test Java 4.x`

It make no sense to create fix-commit for a commit from the same PR.

File: test-cases/longevity/longevity-nosqlbench-3h.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 test_duration: 300
 
 stress_cmd: [
-    "nosqlbench cql-tabular2 rampup-cycles=1M main-cycles=100M write_ratio=5 read_ratio=5 async=300 --progress console:5s rf=3 partsize=5000 driver=cql-scylla -v"
-  ]
+    "nosqlbench cql_tabular rampup-cycles=1M main-cycles=100M write_cl=LOCAL_QUORUM read_cl=LOCAL_QUORUM rf=1 partsize=1000000 partcount=1000 driver=cqld4 -v --progress console:5s",
```
[the remark this answers is not in the document]
Comment (fruch) [reply test-cases/longevity/longevity-nosqlbench-3h.yaml:4]: you are right, in general we should be using rf=3

the main goal of this PR was to prove we have a working latest nosqlbench on top of scylla driver
later we'll gonna define the set of cases we'll need for java driver 4.x CI, and would then we'll need to be more precise on the exact parameters (all of them, not just RF)

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**